README.md

```diff
@@ -71,8 +71,93 @@ one file, `original-specification-example.foo` alone.
     }
 ]
 ```
+
+### Group and Migration Ordering
+
+#### Groups
 Migration files contain groups of migrations. Each group is executed in its entirety before moving on to the next. Groups are executed in the order in which they appear in the migration file. 
 
+Example of ordered groups
+```json
+[ 
+    {
+        "group": "foo",
+        "type": "ordered",
+        "migrations": [
+            {
+                "name": "migration_1"        
+            }
+        ]
+    }, 
+    {
+        "group": "bar",
+        "type": "ordered",
+        "migrations": [
+            {
+                "name": "migration_2"        
+            }
+        ]
+    }
+ ]
+```
+
+#### Migrations
+Migrations within a group can be ordered manually or by version. 
+
+Example of manually ordered migrations
+```json
+[ 
+      {
+         "group": "foo",
+         "type": "ordered",
+         "migrations": [
+            {
+                  "name": "migration_1"        
+            },
+            {
+                  "name": "migration_2" 
+           },
+           {
+                 "name": "migration_3" 
+           },
+           {
+                 "name": "migration_4"
+           }
+        ]
+    }
+ ]
+```
+If Baton is executed, migrations would be ordered in which they are specified (migration_1, migration_2, migration_3, migration_4)
+
+Example of version ordered migrations
+```json
+[ 
+      {
+         "group": "bar",
+         "type": "versioned",
+         "migrations": [
+            {
+                  "name": "migration_1", 
+                  "version": "3.0.0"     
+            },
+            {
+                  "name": "migration_2", 
+                  "version": "2.0.0" 
+           },
+           {
+                 "name": "migration_3",
+                 "version": "1.0.0" 
+           },
+           {
+                 "name": "migration_4", 
+                 "version": "5.0.0" 
+           }
+        ]
+    }
+ ]
+```
+If Baton is executed, migrations would be ordered by ascending version number (migration_3, migration_2, migration_1, migration_4)
```

Comment:
Re-wording suggestion:

If Baton is executed on a versioned migration json, then migrations would be processed by the version number in ascending order (migration_3, migration_2, migration_1, migration_4)

Reply:
Fixed.